packages/form-footer/src/FormFooter.tsx

```diff
@@ -0,0 +1,184 @@
+import React, { useMemo } from 'react';
+import Banner from '@leafygreen-ui/banner';
+import Button from '@leafygreen-ui/button';
+import { css, cx } from '@leafygreen-ui/emotion';
+import { uiColors } from '@leafygreen-ui/palette';
+import { isComponentType } from '@leafygreen-ui/lib';
+import { once } from 'lodash';
+
+const footerStyle = (sticky: boolean) => {
+  return css`
+    position: ${sticky ? 'fixed' : 'static'};
+    bottom: ${sticky ? '0px' : 'unset'};
+    left: ${sticky ? '0px' : 'unset'};
+    min-height: 92px;
+    width: 100%;
+    padding: 24px;
+    border: 1px solid ${uiColors.gray.light2};
+    box-shadow: 0px -4px 4px 0px rgba(6, 22, 33, 0.1);
+  `;
+};
+
+const contentStyle = css`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin: 0 auto;
+  gap: 8px;
+
+  > button {
+    white-space: nowrap;
+  }
+`;
+
+const flexSpan = css`
+  display: inline-flex;
+  gap: inherit;
+`;
+
+const bannerStyle = css`
+  flex-grow: 0;
+  min-height: unset;
+  padding-block: 7px;
```

Comment:
Why are we changing thee left/right padding of the banner?

Reply:
`padding-block` refers to top/bottom. We're changing it since the banner and button are different heights, re [this thread](https://mongodb.slack.com/archives/G01500NFVPS/p1635181579001300)

Reply:
Whoops, you're right. Sometimes it's applied on the left and right based on other CSS properties, but not in this case. Sounds good 👍